scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -100,6 +100,10 @@ def net_config(args):
     return get_config(values, os.path.join(TEMPLATE_DIR, "admin-net.xml"))
 
 
+def merge_dicts(d1, d2):
+    return dict(it.chain(d1.items(), d2.items()))
```

Comment:
@dguitarbite not correct, because you are mutating d1 (side effects) and returning d2 that remains the same. This `merge_dicts` is correct.

Another option is

```python
tmp = d1.copy()
tmp.update(d2)
return tmp
```

But I do not see the reason to change the code.